frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -287,7 +288,10 @@ class Worksheet extends React.Component {
         // Refreshes the checkbox after commands
         // If the action failed, the check will persist
         let force_delete = cmd === 'rm' && this.state.forceDelete ? '--force' : null;
-        this.setState({ updating: true });
+        this.setState({
+            updating: true,
```

Comment:
Note that the broad `updating` state is used outside of command execution, so I didn't want to rely on that state for this particular logic. The `executingCommand` state was introduced to specifically represent updates during command execution.